packages/browser-runtime-core/src/interpreter/preprocessor/inject-last-expression-callback.ts

```diff
@@ -1,21 +1,22 @@
 import template from '@babel/template';
+import type { File, ExpressionStatement, ClassDeclaration, FunctionDeclaration } from '@babel/types';
 
-export function injectLastExpressionCallback(callbackFunctionName, ast): object {
-  const capture = template(`${callbackFunctionName}(%%expression%%)`);
+export function injectLastExpressionCallback(callbackFunctionName: string, ast: File): File {
+  const capture = template.statement(`${callbackFunctionName}(%%expression%%)`);
```

Comment:
`template.statement` is a bit nicer here than `template` because the latter automatically switches between a single `Statement` or an array of them. We’re only dealing with single statements here and can be explicit about that.